packages/core/src/presentation/forms/validation/validators.ts

```diff
@@ -9,13 +9,31 @@ const VALID_EMAIL_REGEX = new RegExp(
   '^(([^<>()\\[\\]\\\\.,;:\\s@"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@"]+)*)|(".+"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$',
 );
 
+const VALID_URL = new RegExp('^https?://.+$');
```

Comment:
can you point some examples how exactly these placeholders/regex patterns can be configurable in `settings.js`, 
and I think these are fixed values do you see any need of making these configurable.

The URL here is a single Message Broker URL where the CDEvents should be delivered.

Reply:
The need is as I described in the first comment - operators may wish to only allow just one or a few URLs here, or limit that to a subset of domains.  

Exfiltration of CDEvents can be a serious security problem, and it would be useful to have an additional guard on the frontend to have that experience align with any guards on the backend.

Reply:
As an example, the code in here would first check for a value from the global `SETTINGS` object, and fall back to a reasonable default if not set.  

```
import { SETTINGS } from '../config';
...
var pattern = SETTINGS?.cdevents?.validUrlPattern ?? '^https?://.+$';
const VALID_URL = new RegExp('^https?://.+$');
...
```

Or something along those lines.  The import path will vary based on where this file is relative to that in the Deck source tree.